production.ini.in

```diff
@@ -38,6 +38,7 @@ sqlalchemy.lubis.url = postgresql://${vars:dbhost}:${vars:dbport}/lubis
 mako.directories = chsdi:templates
 
 data_path = /var/local/cartoweb/
+zadara_downloads = /var/local/cartoweb/downloads/
```

Comment:
It's just to avoid duplication, as this directory is also used in the apache configuration. So when adapting, you need to remember to adapt in 2 places. In that sense, it's more like `print_temp_dir`, which also doesn't change but is defined in the vars section.